client/finality-grandpa/src/communication/mod.rs

```diff
@@ -134,7 +139,18 @@ pub(crate) struct NetworkBridge<B: BlockT, N: Network<B>> {
 	service: N,
 	gossip_engine: GossipEngine<B>,
 	validator: Arc<GossipValidator<B>>,
+
+	/// Sender side of the neighbor packet channel.
+	///
+	/// Packets send into this channel are processed by the `NeighborPacketWorker` and passed on to
+	/// the underlying `GossipEngine`.
 	neighbor_sender: periodic::NeighborPacketSender<B>,
```

Comment:
```suggestion
	neighbor_packet_sender: periodic::NeighborPacketSender<B>,
```

Happy to do this change, but rather as a follow up to reduce the noise within this pull request.